pkg/image/registryclient/client.go

```diff
@@ -35,6 +37,11 @@ type RepositoryRetriever interface {
 	Repository(ctx context.Context, registry *url.URL, repoName string, insecure bool) (distribution.Repository, error)
 }
 
+// ImageSource
+type AdditionalImageSources interface {
+	AlternativeSources(image imagereference.DockerImageReference) ([]imagereference.DockerImageReference, error)
```

Comment:
Just `Alternative`, with the field being same as interface, you'll be getting a nice `ImageSources.Alternative` call

Reply:
updated, done